src/Razor/src/Microsoft.VisualStudio.RazorExtension/.gitignore

```diff
@@ -0,0 +1 @@
+xlf/
```

Comment:
Is it worth adding this to the top level `.gitignore`?

Reply:
I didn't understand what that folder is well enough to make that determination (maybe we have other spots where we have it checked in?) so I went narrow, but if anyone knows better I would certainly prefer that it be at the top level if possible.